test/gurka/test_graphfilter.cc

```diff
@@ -0,0 +1,601 @@
+#include "baldr/graphreader.h"
+#include "mjolnir/util.h"
+
+#include "gurka.h"
+#include "test/test.h"
+
+using namespace valhalla;
+using namespace valhalla::baldr;
+using namespace valhalla::gurka;
+
+const std::string work_dir = {VALHALLA_BUILD_DIR "test/data/gurka_graphfilter"};
+
+TEST(Standalone, SimpleFilter) {
+  constexpr double gridsize_metres = 10;
+
+  const std::string ascii_map = R"(
+
+             M              P         R              W    Y
+             |              |         |              |    |
+             |              |         |              |    |
+        A----B----C----D----E----F----G----H----I----J----K----L
+                  |    |         |         |    |         |
+                  |    |         Q----S----T    |         |
+                  N    O              |         V         X
+                                      |
+                                      U 
```

Comment:
to be fully honest, it feels like the maps in here are way more detailed than they need to be, which would add a lot of cognitive overhead if we ever have to manipulate them again.

also, there wasn't ever a graph filter test? good that it's added now:)

Reply:
Yeah I always find real world examples and port them over to gurka.  I have been burned too many times by creating "fake" tests.  This is why they seem so complex.  You can use LL to see the location from where I created the test from.